src/ui/pages/Notifications/NotificationItem.tsx

```diff
@@ -51,47 +76,49 @@ const NotificationItem = ({
     };
 
     fetchData();
-  }, [item]);
+  }, [fetchNotificationLabel, item]);
 
-  const notificationLabel = async (
-    item: KeriaNotification,
-    multisigIcpDetails?: MultiSigIcpRequestDetails
-  ) => {
+  const referIcon = (item: KeriaNotification) => {
     switch (item.a.r) {
     case NotificationRoute.ExnIpexGrant:
-      return i18next.t("notifications.tab.labels.exnipexgrant", {
-        connection: connection,
-      });
-    case NotificationRoute.MultiSigIcp:
-      return i18next.t("notifications.tab.labels.multisigicp", {
-        connection: multisigIcpDetails?.sender?.label,
-      });
     case NotificationRoute.ExnIpexApply:
-      return i18next.t("notifications.tab.labels.exnipexapply", {
-        connection: connection,
-      });
+      return fingerPrintOutline;
+    case NotificationRoute.MultiSigIcp:
+      return MultisignReferIcon;
```

Comment:
These icons need to be swapped, they are the other way around.

![11072024155807](https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/51099606-3110-43b5-877b-447fceaa43ea)